spec/core_functions/meta/mixin_exists.hrx

```diff
@@ -0,0 +1,384 @@
+<===> same_module/global/input.scss
+@mixin global-mixin() {}
+
+a {b: mixin-exists(global-mixin)}
+
+<===> same_module/global/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/local/input.scss
+a {
+  @mixin local-mixin() {}
+  b: mixin-exists(local-mixin);
+}
+
+<===> same_module/local/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/through_import/input.scss
+@import "other";
+a {b: mixin-exists(global-mixin)}
+
+<===> same_module/through_import/other.scss
+@mixin global-mixin() {}
+
+<===> same_module/through_import/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> same_module/non_existent/input.scss
+a {
+  b: mixin-exists(non-existent);
+}
+
+<===> same_module/non_existent/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> different_module/options.yml
+---
+:todo:
+- sass/libsass#2807
+- sass/dart-sass#554
+
+<===>
+================================================================================
+<===> different_module/undefined/input.scss
+@use "sass:color";
+a {b: mixin-exists("c", "color")}
+
+<===> different_module/undefined/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> different_module/defined/input.scss
+@use "other";
+a {b: mixin-exists("c", "color")}
+
+<===> different_module/defined/_other.scss
+@mixin c() {}
+
+<===> different_module/defined/error
+Error: There is no module with the namespace "color".
+  ,
+2 | a {b: mixin-exists("c", "color")}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 2:7  root stylesheet
+
+<===>
+================================================================================
+<===> different_module/chosen_prefix/input.scss
+@use "other" as a;
+b {c: mixin-exists("d", "a")}
+
+<===> different_module/chosen_prefix/_other.scss
+@mixin d() {}
+
+<===> different_module/chosen_prefix/output.css
+b {
+  c: true;
+}
+
+<===>
+================================================================================
+<===> different_module/through_use/options.yml
```

Comment:
Unnecessary given the options.yml in the outer directory? (same below)

Reply:
Done.